python/src/scipp/plot/plot.py

```diff
@@ -83,14 +83,7 @@ def plot(scipp_obj,
     # {number_of_dimensions, Dataset, axes, line_parameters}.
     tobeplotted = dict()
     for name, var in sorted(inventory.items()):
-
-        if sc.contains_events(var) and bins is None:
-            raise RuntimeError("The `bins` argument must be specified when "
-                               "plotting event data.")
-
         ndims = len(var.dims)
-        if bins is not None and sc.contains_events(var):
-            ndims += 1
```

Comment:
@nvaytet See also here. I suppose we will eventually want special handling for buckets. Is it fine to just leave it out in this PR?

Reply:
I would say that if you can get the plot to show up and it works as you want, then it's fine.
We will definitely need to add handling of buckets in the plots, so fine to leave it out for now.